app/assets/javascripts/controllers/live_metrics/util/metrics-parse-url-factroy.js

```diff
@@ -4,10 +4,12 @@ angular.module('miq.util').factory('metricsParseUrlFactory', function() {
     var pathname = $window.location.pathname.replace(/\/$/, '');
     dash.providerId = '/' + (/^\/[^\/]+\/([r\d]+)$/.exec(pathname)[1]);
 
-    // TODO: get this values from GET/POST values ?
+    // parse window's url
+    dash.params = (new URL($window.location)).searchParams;
+
     dash.tenantList = [];
-    dash.minBucketDurationInSecondes = 20 * 60;
-    dash.max_metrics = 10000;
-    dash.items_per_page = 10;
+    dash.minBucketDurationInSecondes = parseInt(dash.params.get("bucket")) || (20 * 60);
```

Comment:
👍 yes I'm working on that ...

@himdel I get:
`TypeError: undefined is not an object (evaluating 'dash.params.get') (line 11)`

after I added 
```
var URL = function() { 
      return {
        searchParams: { get: function() { return null; } }
      };
    };
```
to the spec :-) what am I missing ?